Add documentation for Backup integration (#21612)
home-assistant/home-assistant.github.io: source/_integrations/backup.markdown

---
title: Backup

> description: System Health allow you to create backups on Core and Container installations.

Reviewer: ```suggestion
description: Allow creating backups of container and core installations.
```

ha_category:
  - Other

> ha_release: 2022.3

Author: Will not make 2022.3, version number needs adjustment

ha_quality_scale: internal
ha_domain: backup
---

> The Backup integration allow you to create and download backups for your Home Assistant Core and Home Assistant Container installation, this backup file can be used if you migrate to Home Assistant Operating System.

Reviewer: ```suggestion
The Backup integration allow you to create and download backups for your Home Assistant Core and Home Assistant Container installations. This backup file can be used if you migrate to Home Assistant Operating System.
```

<div class="note">

> If you use Home Assistant Operating System or Home Assistant Supervised, this integration is not for you, you already have [Backup functionaloty with the Supervisor](/common-tasks/os/#backups).

Reviewer: ```suggestion
If you use Home Assistant Operating System or Home Assistant Supervised, [back up functionality is already built-in]](/common-tasks/os/#backups).
```
